cmd/gossamer/flags.go

```diff
@@ -100,7 +100,7 @@ var (
 	}
 	// BasePathFlag data directory for node
 	BasePathFlag = cli.StringFlag{
-		Name:  "basepath",
+		Name:  "base-path,basepath", // base-path is argument used by polkadot node
```

Comment:
Should we just have breaking changes and change these flags to match polkadot? @danforbes ?

Reply:
I'm fine with either approach.